cirrus/google_cloud/manager.py

```diff
@@ -35,6 +35,7 @@
 GOOGLE_IAM_API_URL = "https://iam.googleapis.com/v1/"
 GOOGLE_CLOUD_RESOURCE_URL = "https://cloudresourcemanager.googleapis.com/v1/"
 GOOGLE_DIRECTORY_API_URL = "https://www.googleapis.com/admin/directory/v1/"
+GOOGLE_LOGGING_EMAIL = 'cloud-storage-analytics@google.com'
```

Comment:
Seems like GitHub is being weird... this isn't related to your PR and should be in master but it's showing up as a new addition. Did you merge/rebase master into your branch?

Reply:
I did rebase on the master